cluster/manifests/kube-metrics-adapter/deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       serviceAccountName: custom-metrics-apiserver
       containers:
       - name: kube-metrics-adapter
-        image: registry.opensource.zalan.do/teapot/kube-metrics-adapter:v0.1.17
+        image: registry.opensource.zalan.do/teapot/kube-metrics-adapter:v0.1.17-6-g9d8359b
```

Comment:
Would be nice to tag a new release `v0.1.18` now that the bug is fixed. Not only for us, but also for our open source users. :)